src/proj.h

```diff
@@ -269,6 +282,42 @@ struct PJ_OBS {
     unsigned int flags;  /* additional data, intended for flags */
 };
 
+struct PJ_INFO {
+    char        release[64];        /* Release info. Version + date         */
+    char        version[64];        /* Full version number                  */
+    int         major;              /* Major release number                 */
+    int         minor;              /* Minor release number                 */
+    int         patch;              /* Patch level                          */
+    char        searchpath[512];    /* Paths where init and grid files are  */
+                                    /* looked for. Paths are separated by   */
+                                    /* semi-colons.                         */
+};
+
+struct PJ_PROJ_INFO {
+    char        id[16];             /* Name of the projection in question           */
+    char        description[128];   /* Description of the projection                */
+    char        definition[512];    /* Projection definition                        */
+    int         has_inverse;        /* 1 if an inverse mapping exists, 0 otherwise  */
+};
+
+struct PJ_GRID_INFO {
+    char        gridname[32];       /* name of grid                         */
+    char        filename[128];      /* full path to grid                    */
```

Comment:
I'll change the length of filename to `MAX_PATH` in both `PJ_INIT_INFO` and `PJ_GRID_INFO`. That should be enough for all but the ridiculous cases. I'll just do it as `char filename[260]` since I don't want to clutter the header too much with a bunch of #defines at this time.

In `PJ_INFO` the searchpath is 512 at the moment. I think it is highly unlikely that it will be maxed out, but of course it could be changed as well. Maybe to 1024? Opinions?